packages/base-styles/_mixins.scss

```diff
@@ -489,12 +489,9 @@
 	/* stylelint-enable function-comma-space-after */
 }
 
-@mixin custom-scrollbars-on-hover() {
+@mixin custom-scrollbars-on-hover($track-color: #1e1e1e, $handle-color: #757575) {
```

Comment:
Nice, we have a mixin for this! It might be good to include the following query in that mixin, so that scrolling is always enabled on mobile, otherwise you have to tap the container before the scrollbar appears: 

```
// Always enable scrollbar on Mobile devices.
@media (hover: none) {
	.scrollable {
	  visibility: visible;
	}
}
```